site/docs/numbers/hostingNumbers.mdx

```diff
@@ -95,233 +122,179 @@ Content-Type: application/xml; charset=utf-8
-   <SiteId>14419</SiteId>
-   <Subscriber>
-      <Name>BANDWDITH</Name>
-      <ServiceAddress>
-         <HouseNumber>900</HouseNumber>
-         <StreetName>Main Campus Dr.</StreetName>
-         <City>Raleigh</City>
-         <StateCode>NC</StateCode>
-         <Zip>27606</Zip>
-         <County>Wake</County>
-         <Country>United States</Country>
-         <AddressType>Service</AddressType>
-      </ServiceAddress>
-   </Subscriber>
-   <LoaAuthorizingPerson>Mark Macchioni</LoaAuthorizingPerson>
-   <TelephoneNumbers>
-      <TelephoneNumber>9195041533</TelephoneNumber>
-   </TelephoneNumbers>
-   <ProcessingStatus>COMPLETE</ProcessingStatus>
-   <Errors />
-   <SipPeerId>521967</SipPeerId>
-</ImportTnOrder>
+<ImportTnOrderResponse>
+   <ImportTnOrder>
+      <CustomerOrderId>marktestorders</CustomerOrderId>
+      <OrderCreateDate>2018-01-20T02:59:54.000Z</OrderCreateDate>
+      <AccountId>9900012</AccountId>
+      <CreatedByUser>{username}</CreatedByUser>
+      <OrderId>b05de7e6-0cab-4c83-81bb-9379cba8efd0</OrderId>
+      <LastModifiedDate>2018-01-20T02:59:54.000Z</LastModifiedDate>
+      <SiteId>202</SiteId>
+      <SipPeerId>520565</SipPeerId>
+      <Subscriber>
+         <Name>ABC Inc.</Name>
+         <ServiceAddress>
+            <HouseNumber>11235</HouseNumber>
+            <StreetName>Back</StreetName>
+            <City>Denver</City>
+            <StateCode>CO</StateCode>
+            <Zip>27541</Zip>
+            <County>Canyon</County>
+         </ServiceAddress>
+      </Subscriber>
+      <LoaAuthorizingPerson>markmacc</LoaAuthorizingPerson>
+      <TelephoneNumbers>
+         <TelephoneNumber>3032281000</TelephoneNumber>
+         <TelephoneNumber>3032280004</TelephoneNumber>
+      </TelephoneNumbers>
+      <ProcessingStatus>PROCESSING</ProcessingStatus>
+      <Errors />
+   </ImportTnOrder>
+</ImportTnOrderResponse>
 ```
 
-</TabItem>
-</Tabs>
+## Fetch Order Status
 
-## Upload Letter of Authorization (LOA)
+Since the Import TN order is asynchronous, Bandwidth recommends configuring your account with a subscription instead of polling the order resource for `OrderStatus`.
+Order processing times can vary and are not guaranteed, so bandwidth does __not__ recommend setting a timeout on waiting for an order to show either `COMPLETE` or `FAILED` status, but instead relying on a webhook from an orders subscription.
+Please follow the [How to setup Notification Webhook](/docs/numbers/webhooks/orderWebhook) guide for more information.
 
-For **completed** orders, Bandwidth requires a completed Subscriber "Letter of Authorization" (LOA) from the phone number user.  The LOA file is used in the case there is a dispute to ensure the phone number had permission to be enabled for hosted messaging for Bandwidth.
+Otherwise, you can make <Highlight color="#00bf8c">GET</Highlight> request to [Fetch Import TN Order status API](../../../../apis/numbers/#tag/Import-Tn/operation/ReadImportTnOrder) endpoint to poll the order status.
```

Comment:
```suggestion
Otherwise, you can make a <Highlight color="#00bf8c">GET</Highlight> request to [Fetch Import TN Order status API](../../../../apis/numbers/#tag/Import-Tn/operation/ReadImportTnOrder) endpoint to poll the order status.
```